### ISSUE TITLE
About mask training dataset

### ISSUE THREAD

**Issue by haolin512900** [NVIDIA/partialconv#30]
Thank you for your great project! 
        I am sorry,Because I don't found the website of the mask training dataset from the paper,Do you provide the website? If you do I will  appreciate it!
        Looking forward to your reply.

**Comment by liuguilin1225** [comment]
The mask dataset can be found at: https://nv-adlr.github.io/publication/partialconv-inpainting

The training set is the raw mask dataset and you need to apply the augmentation on top of it by yourself. Augmentation details are included in the linked page.
The testing set is the already augmented and well-organized according to the mask ratio.  (If dropbox doesn't work, you can also access the testing set at http://masc.cs.gmu.edu/wiki/uploads/partialconv/mask.zip )

**Comment by haolin512900** [comment]
Thank you ! 
good luck

**Comment by haolin512900** [comment]
> The mask dataset can be found at: https://nv-adlr.github.io/publication/partialconv-inpainting
> 
> The training set is the raw mask dataset and you need to apply the augmentation on top of it by yourself. Augmentation details are included in the linked page.
> The testing set is the already augmented and well-organized according to the mask ratio. (If dropbox doesn't work, you can also access the testing set at http://masc.cs.gmu.edu/wiki/uploads/partialconv/mask.zip )

thank you ! good luck to you

**Comment by haolin512900** [comment]
Hi ,Can you shared the link of the training set?(Because the dropbox doesn't work)😂

**Comment by haolin512900** [comment]
> The mask dataset can be found at: https://nv-adlr.github.io/publication/partialconv-inpainting
> 
> The training set is the raw mask dataset and you need to apply the augmentation on top of it by yourself. Augmentation details are included in the linked page.
> The testing set is the already augmented and well-organized according to the mask ratio. (If dropbox doesn't work, you can also access the testing set at http://masc.cs.gmu.edu/wiki/uploads/partialconv/mask.zip )

Hi ,Can you shared the link of the training set?(Because the dropbox doesn't work)😂
